stackit/internal/utils/utils.go

```diff
@@ -76,3 +76,17 @@ func SimplifyBackupSchedule(schedule string) string {
 	})
 	return simplifiedSchedule
 }
+
+func PossibleValuesDocumentation(values []string) string {
+	if len(values) == 0 {
+		return ""
+	}
+	return "Possible values are: " + strings.Join(quoteValues(values), ", ") + "."
```

Comment:
`Possible values` doesn't sound strict enough, especially if we are then verifying that values are exactly one of them.

Something like `Value must be one of the following:`

Reply:
`Supported values:` sounds good